modules/apps/27-interchain-accounts/host/keeper/keeper_test.go

```diff
@@ -136,7 +136,7 @@ func (suite *KeeperTestSuite) TestNewKeeper() {
 	testCases := []struct {
 		name          string
 		instantiateFn func()
-		expPass       bool
+		exxPanicMsg   string
```

Comment:
```suggestion
		expPanicMsg   string
```

Reply:
We could also just use `panicMsg`, since you use `errMsg` in other tests, but it's just a nit :)